index.js

```diff
@@ -53,15 +64,10 @@ module.exports = (nextConfig = {}) => ({
     } else if (!options.isServer) {
       // Only run once for the client build.
       config.plugins.push(
-        new CleanWebpackPlugin(['precache-manifest.*.js'], { root: config.output.path, verbose: false }),
+        // Workbox uses Webpack's asset manifest to generate the SW's pre-cache manifest, so we need
+        // to copy the app's assets into the Webpack context so those are picked up.
+        new CopyWebpackPlugin([{ from: `${join(cwd(), nextAssetPrefixDir)}/**/*` }]),
         generateSw ? new GenerateSW({ ...workboxOpts }) : new InjectManifest({ ...workboxOpts }),
-        new InlineNextPrecacheManifestPlugin({
```

Comment:
No longer necessary, the new `exclude/modifyURLPrefix` options handle all of this now.